docs/usermanual.md

```diff
@@ -116,10 +116,36 @@ tools available online like [this one](https://string-functions.com/string-hex.a
 
 It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with
 `payloadType` `ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure
-attributes and `id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used,
-as well as `actionType`.
+attributes. The entity `id` and `type` are also include as attributes `ngsi_id` and `ngsi_type` (both with attribute
+type `string`). Destination entity `id` and entity `type` from device/group configuration provisioned are used,
+as well as `actionType`. It is possible to use the same entity `id` that the original one by using `entityNameExp` 
+at [device group provision](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#config-group-datamodel)
 
-Examples of these `ngsiv2` payloads are the following ones:
+Taking as an example the follwing entity as measure:
+
+```json
+{
+    "id": "MyEntityId1",
+    "type": "MyEntityType1",
+    "attr1": { "type": "Text", "value": "MyAttr1Value"}
+}
+```
+
+It would generate an entity like the following one
+
+```json
+{
+    "id":"MyProvisionID",
+    "type":"MyProvisionType",
+    "attr1": { "type": "Text", "value": "MyAttr1Value"},
+    "ngsi_id": {"type":"string","value":"MyEntityId1"},
```

Comment:
This ngsi_id should be appear only in ngsi payload examples

Reply:
Modified to be more precise here:https://github.com/telefonicaid/iotagent-json/pull/806/commits/e07b04f561fa98689626fd2eab98386c3b8e7582